### PR TITLE
Update snippets after Node Code Snippet audit

### PR DESCRIPTION
**Pull request by AlexLakatos**

### CODE REVIEW

File: account/create-secret.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ require('dotenv').config({path: __dirname + '/../.env'})
 
 const NEXMO_API_KEY = process.env.NEXMO_API_KEY
 const NEXMO_API_SECRET = process.env.NEXMO_API_SECRET
+const NEW_SECRET = process.env.NEW_SECRET
```
Comment (lornajane) [inline account/create-secret.js:5]: Should this be in the `.env.example` file also?

File: voice/play-tts-into-call.js [hunk 2]
```diff
@@ -7,6 +7,8 @@ const NEXMO_API_SECRET = process.env.NEXMO_API_SECRET;
 const NEXMO_PRIVATE_KEY = __dirname +"/../"+ process.env.NEXMO_PRIVATE_KEY;
 const NEXMO_APPLICATION_ID = process.env.NEXMO_APPLICATION_ID;
 
+const UUID = process.env.UUID;
```
Comment (lornajane) [inline voice/play-tts-into-call.js:10]: Should this `UUID` be used in the `nexmo.calls.talk.start()` instead of `CALL_UUID`?

**Review by lornajane — COMMENTED** [review]
Couple of minor nitpicks but everything seems basically sound!

File: account/create-secret.js (hunk 1 above)
Comment (AlexLakatos) [reply account/create-secret.js:5]: Yes it should, thanks for catching this.

File: voice/play-tts-into-call.js (hunk 2 above)
Comment (AlexLakatos) [reply voice/play-tts-into-call.js:10]: Yes it should, thanks for catching this.

**Review by lornajane — APPROVED** [review]
This is in great shape!

**Pull request merged**